src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -882,4 +882,7 @@ Do you want to continue?</value>
   <data name="WRN_UnableToLoadAnalyzer" xml:space="preserve">
     <value>Unable to load Analyzer assembly {0}: {1}</value>
   </data>
+  <data name="Make_method_synchronous" xml:space="preserve">
```

Comment:
i'm not a fan of that. it makes reading stuff like "ThisIs0TheError1MessageWith2PlaceHolders3InIt".  It also means your casing in the field name doesn't match the casing of your actual English string.  It's much cleaner to have "This_is_0_the_error_1_message_with_2_place_holders_3_in_it".  This is *much* more readable.  Makes it much clearer where the placeholders are.  And doesn't unnecessarily capitalize items which aren't going to actually be in capitals in English.